templates/dashboard.html

```diff
@@ -180,6 +180,15 @@ <h3>{{gettext("Wallets excluded from circulating supply")}}</h3>
               </a>
             </div>
 
+            <div class="reserved-wallet">
+              <div class="wallet-name">{{gettext("Lukewarm Wallet")}}</div>
```

Comment:
Is "Lukewarm wallet" how we want to officially and publicly call this wallet?

Reply:
No, let's stick with calling it "Distribution Staging" as it's labeled on CoinGecko.

Reply:
Actually, we already have that wallet listed. My bad, I didn't check the address before adding. 😅